src/applications/disability-benefits/all-claims/utils/on-submit/purge-toxic-exposure-data.js

```diff
@@ -141,32 +153,43 @@ const purgeExposureDetails = (toxicExposure, exposureType, mapping) => {
  *
  * This function handles scenarios where users:
  * - Start filling sections then deselect them (leaving orphaned details)
- * - Select "none" for conditions (complete opt-out)
- * - Have no conditions selected (empty/missing/all false - removes all exposure data)
+ * - Select "none" for conditions (complete opt-out - removes all exposure data, keeps only { conditions: { none: true } })
+ * - Have no conditions selected (empty/missing/all false - removes entire toxicExposure object)
  * - Remove all selections but leave partial data behind
  * - Have null exposure fields that need cleanup
+ * - Have orphaned "other" fields (otherHerbicideLocations, specifyOtherExposures) when corresponding 'other' selection not true
  *
  * Key behaviors:
  * - Preserves unknown/unrecognized fields (forward compatibility)
+ * - Keeps false values in selection objects (conditions, exposures) for backend visibility
```

Comment:
Unless in the case of a user selecting "none" for conditions or no conditions are selected (empty/missing/false for all).